captura/test_views.py

```diff
@@ -0,0 +1,60 @@
+from django.core.urlresolvers import reverse
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from splinter import Browser
+from django.contrib.auth.models import User
+from perfiles_usuario.utils import CAPTURISTA_GROUP
```

Comment:
the order of imports is wrong. check two scoops section 1.3. Local imports should be at the end.